Sources/CollectionViewDriver.swift

```diff
@@ -83,14 +83,14 @@ public final class CollectionViewDriver: NSObject {
     /// Thus, the caller is responsible for retaining and keeping alive the `cellEventCoordinator`
     /// for the entire lifetime of the driver.
     public init(view: UICollectionView,
-                layout: UICollectionViewCompositionalLayout,
+                layout: UICollectionViewCompositionalLayout? = nil,
```

Comment:
To summarize from my TED Talk comment: let's just remove this completely.

Don't forget to remove the docs too. 😊